testing/integration_test.go

```diff
@@ -33,6 +33,8 @@ import (
 )
 
 var (
+	repo   = flag.String("repo", "", "git repo to test")
+	branch = flag.String("branch", "", "git branch to test")
```

Comment:
I thought just knowing the commit should be enough. Why do you also need branch?